crates/driver/src/boundary/settlement.rs

```diff
@@ -262,7 +262,7 @@ fn to_boundary_order(order: &competition::Order) -> Order {
             buy_token: order.buy.token.into(),
             sell_amount: order.sell.amount.into(),
             buy_amount: order.buy.amount.into(),
-            fee_amount: order.user_fee.into(),
+            fee_amount: 0.into(),
```

Comment:
I think commenting here that `fee` is guaranteed to be `0` and therefore omitted from the remaining source code but is still required to correctly encode the order here would be useful.
Actually given that solvers can encode their own solutions in the future we probably need to mention this somewhere in the docs.

Reply:
Added comment in the code.